concepts/channels/channel_normalization.md

```diff
@@ -0,0 +1,86 @@
+# Telemetry Channel Behavior
+
+In every ping there are two channels:
+- App Update Channel
+- Normalized Channel 
+
+## Expected Channels
+The traditional channels we expect are:
+- `release`
+- `beta`
+- `aurora` (this is `dev-edition`, and is just a beta repack)
+- `nightly`
+- `esr`
+
+## App Update Channel
+This is the channel reported by the application directly. This could really be anything, but is usually one of the
+expected release channels listed above.
+
+### Accessing App Update Channel
+
+#### Main Summary
+The field here is called `channel`, e.g.
+```
+SELECT channel
+FROM main_summary
+WHERE submission_date_s3 = '20180823'
```

Comment:
Should we include a `LIMIT 10` here and in queries below, in case someone copy/pastes the query directly?